tests/test_indexes.py

```diff
@@ -81,13 +86,56 @@
 
 test_cases_visualization = []
 
-test_cases = (
+# Custom test cases, a dictionary of {func_str: test_case}
+test_case_custom = {}
+for case in (
     test_cases_nlp
     + test_cases_preprocessing
     + test_cases_representation
     + test_cases_visualization
+):
+    test_case_custom[case[0]] = case
+
+
+# Put functions' name into white list if you want to omit them
+# func_white_list = {
```

Comment:
Remove this and the next few lines? (The comment)

Reply:
Done!